Formula/zookeeper.rb

```diff
@@ -37,6 +41,7 @@ def shim_script(target)
   def default_zk_env
```

Comment:
should probably use the style of https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/boot-clj.rb and also not be a separate function. 

In addition to that the defaults writing should be moved to a `post_install` because currently it'll always overwrite the user settings.

Reply:
I made some changes.